tinkerpop3/src/main/java/io/shiftleft/overflowdb/ReferenceManager.java

```diff
@@ -42,9 +42,9 @@ public void applyBackpressureMaybe() {
     synchronized (backPressureSyncObject) {
       while (clearingProcessCount > 0) {
         try {
-          logger.trace("wait until ref clearing completed");
+          if (logger.isTraceEnabled()) logger.trace("wait until ref clearing completed");
```

Comment:
Since you didn't have this everywhere, does it make a difference, performance-wise? Plus there's nothing getting concatenated here, so it's not exactly avoiding much work?

Reply:
i wasn't sure, and didn't want to start jmh for this. maybe it doesn't make a difference. idk. 
if you're sure that it's not different let me know and i'll remove it again. otherwise i'll just play it 'safe'